### PR TITLE
Add RemoteHamRadio version 2.2.1

### PR DESCRIPTION
**Pull request by macauley**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask <cask>` is error-free.
- [X] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [X] `brew audit --new-cask <cask>` worked successfully.
- [X] `brew install --cask <cask>` worked successfully.
- [X] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/remotehamradio.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask "remotehamradio" do
+  version "2.2.1"
+  sha256 "f042a12c30227723eebdaaaa348ebb6b3ade4dde4126beac2514fd9a7dc10499"
+
+  url "https://s3.amazonaws.com/rhr-electron/desktop-releases/RemoteHamRadio-#{version}-mac.zip", verified: "s3.amazonaws.com/rhr-electron"
```
Comment (neutric) [inline Casks/remotehamradio.rb:5]: Please put `verified` on a new line with 4 space indentation vs. `url` to be consistent with the rest of the casks.

File: Casks/remotehamradio.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  name "RemoteHamRadio"
+  desc "Desktop console app for RemoteHamRadio service"
+  homepage "https://www.remotehamradio.com/"
+
+  livecheck do
+    url "http://update.remotehamradio.com/desktop/download?platform=Macintosh"
+    strategy :page_match
```
Comment (neutric) [inline Casks/remotehamradio.rb:12]: `strategy :page_match` should only be used when explicitly needed. It is the default except for special cases such as GitHub-URLs.

File: Casks/remotehamradio.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    regex(%r{href=.*?/RemoteHamRadio-(\d+(?:\.\d+)*)-mac\.zip}i)
```
Comment (neutric) [inline Casks/remotehamradio.rb:13]:
```
regex(/href=.*?RemoteHamRadio[._-]v?(\d+(?:\.\d+)+)-mac\.zip/i)
```
The examples in the [Cask Cookbook](https://docs.brew.sh/Cask-Cookbook#stanza-livecheck) are not up to date as cask-specific documentation is in the process of being folded into the general Homebrew documentation. This is authoritative: https://docs.brew.sh/Brew-Livecheck#regex-guidelines.

File: Casks/remotehamradio.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  app "RemoteHamRadio.app"
```
Comment (neutric) [inline Casks/remotehamradio.rb:16]: Please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

**Review by neutric — COMMENTED** [review]
Thanks for your contribution to `homebrew-cask`, @macauley! My review is in the code. Take it with a grain of salt as I am not a maintainer.

File: Casks/remotehamradio.rb (hunk 4 above)
Comment (macauley) [reply Casks/remotehamradio.rb:16]: zap stanza added

**Comment by miccal** [comment]
Thank you @macauley.

**Pull request merged**